### PR TITLE
Volumebrowser

### PR DESCRIPTION
**Pull request by XK4MiLX**
**Volume Browser Features:**
upload multiple file
download multiple file
download zipped directory
rename file/folder
craeate directory
remove file/folder
volume storage info
Other UI tweaks

### CODE REVIEW

File: HomeUI/src/views/apps/Management.vue [hunk 1]
```diff
@@ -7146,11 +7754,11 @@ export default {
       }
     },
 
-    cancelDownload() {
-      this.abortToken.cancel('User download cancelled');
-      this.downloaded = '';
-      this.total = '';
-    },
+    // cancelDownload() {
```
Comment (Cabecinha84) [inline HomeUI/src/views/apps/Management.vue:7757]: If not needed should be removed from the code

**Review by Cabecinha84 — CHANGES_REQUESTED** [review]
looking good. Just a small thing that can be deleted.

File: ZelBack/src/routes.js [hunk 2]
```diff
@@ -636,7 +636,7 @@ module.exports = (app, expressWs) => {
     appsService.appendRestoreTask(req, res);
   });
 
-  app.post('/ioutils/fileupload/:fullpath?/:filename?/:appname?', (req, res) => {
```
Comment (TheTrunk) [inline ZelBack/src/routes.js:639]: is this compatible with older versions?

**Review by TheTrunk — CHANGES_REQUESTED** [review]

File: ZelBack/src/routes.js (hunk 2 above)
Comment (XK4MiLX) [reply ZelBack/src/routes.js:639]: no api was changed

Comment (TheTrunk) [reply ZelBack/src/routes.js:639]: But that means its not compatible with older versions of home ui.

Comment (XK4MiLX) [reply ZelBack/src/routes.js:639]: older version using full path as arg now all is done on backend so it is not compatible

**Pull request merged**